app/routes/organen/fracties.js

```diff
@@ -102,7 +102,7 @@ export default class FractiesRoute extends Route {
   }
 
   @action
-  reloadModel() {
+  async reloadModel() {
```

Comment:
Not sure how useful the async is here if you don't await anything.

Reply:
Removed it